packages/core/cli/src/index.ts

```diff
@@ -380,13 +336,6 @@ async function showOfficialPluginList() {
 
   stdin.setRawMode && stdin.setRawMode(true);
   stdin.resume();
-
-  function trimEndingSlash(url: string) {
```

Comment:
i should bring it back somewhere...

Reply:
?

Reply:
i mean i should keep the function for trimming slashes to make URL matching correctly.